src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Serialization/RazorDiagnosticJsonConverter.cs

```diff
@@ -7,7 +7,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 
-namespace Microsoft.VisualStudio.LanguageServices.Razor.Serialization
+namespace Microsoft.CodeAnalysis.Razor.Serialization
```

Comment:
Moved to 1 shared location and removed the duplicated `Razor.LanguageServer.Common/Serialization/RazorDiagnosticJsonConverter.cs`